### PR TITLE
FIX: when setting up multisig, import cosigner via scanning QR with m…

### PR DESCRIPTION
**Pull request by Overtorment**
…nemonic seed

### CODE REVIEW

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/88p032rpgryxfbwfm97xj3fgzg

[download apk](http://filestorage.bluewallet.io:1488/1606763103.apk)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: android/app/build.gradle [hunk 1]
```diff
@@ -136,7 +136,7 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
         versionCode 1
-        versionName "5.6.7"
+        versionName "5.6.8"
```
Comment (Overtorment) [inline android/app/build.gradle:139]: oops i mixed ver bump and feature code in single branch and no one noticed

Comment (marcosrdz) [reply android/app/build.gradle:139]: I did. It made sense to me given that we needed a version bump.
I figured it was Intentional.

**Comment by ncoelho** [comment]
Also saw it, that’s why tagged the previous commit

**Pull request merged**